src/components/data/row/CurrencyRow.tsx

```diff
@@ -62,6 +62,9 @@ const CurrencyRowComponent = (props: Props) => {
   if (showTokenNames && tokenFromId != null) {
     displayCurrencyCode = `${tokenFromId.displayName}`
   }
+  if (token != null) {
```

Comment:
Do not add network names to Wallet List scene. There's not enough space and it was not part of the task

Reply:
The task mentioned "Search wallets list," what was the intended change?